src/javascript/binary/static/viewbalance/viewbalance.data.js

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
I am reimplementing the wheel, reason being that sharing TradeSocket is not a good idea because it get closed every now and then but balance needs to be always open once authorized, currently this is a temporary fix, would move to binary-live-api